crates/host/src/wasmbus/mod.rs

```diff
@@ -1903,6 +1913,10 @@ impl Host {
             ("hostcore.os".into(), OS.into()),
             ("hostcore.osfamily".into(), FAMILY.into()),
         ]);
+
+        let meter = global::meter("wasmcloud-host");
+        let metrics = HostMetrics::new(&meter);
```

Comment:
Nit: it's odd to see these stuck in between the logic for labels. Why not move them down to where they'll be used?

Reply:
Yes, absolutely. Not entirely sure why I stuck them here to begin with, so I've gone ahead and moved them down next to the where the host instance gets initialized.